library/std/src/sys/common/alloc.rs

```diff
@@ -24,7 +24,8 @@ pub const MIN_ALIGN: usize = 8;
     target_arch = "mips64",
     target_arch = "s390x",
     target_arch = "sparc64",
-    target_arch = "riscv64"
+    target_arch = "riscv64",
+    target_arch = "wasm64",
```

Comment:
This isn't something you need to fix in *this* PR, but it feels like these should use `#[cfg(target_pointer_width = "64")]` and similar, and then just list any exceptions to that.

Reply:
If someone forgets to add an exception that may lead to a hard to track down bug.

Reply:
I think this would still be a maintenance improvement even if code directly above this had a list of all target architectures and a compilation error for an unknown architecture.